src/dotnet/Chat.UI.Blazor/Components/MentionList/mention-list.ts

```diff
@@ -0,0 +1,48 @@
+const LogScope: string = 'MentionList';
+
+export class MentionList {
+    private blazorRef: DotNet.DotNetObject;
+    private readonly mentionList: HTMLElement;
+    private mentionListObserver : MutationObserver;
+    private readonly listTop: number;
+    private readonly listBottom: number;
+
+    static create(mentionList: HTMLElement, blazorRef: DotNet.DotNetObject): MentionList {
+        return new MentionList(mentionList, blazorRef);
+    }
+
+    constructor(mentionList: HTMLElement, blazorRef: DotNet.DotNetObject) {
+        this.mentionList = mentionList;
+        this.blazorRef = blazorRef;
+        this.mentionListObserver = new MutationObserver(this.scrollAtCurrentItem);
+        this.mentionListObserver.observe(this.mentionList, {
+            attributes: true,
+            childList: true,
+            subtree: true,
+        })
+        const rect = this.mentionList.getBoundingClientRect();
+        this.listTop = rect.top;
```

Comment:
At creation moment 'hidden' class is applied to mentionList. Are values of listTop and listBottom not equal at that time?